selfdrive/car/hyundai/values.py

```diff
@@ -93,6 +93,7 @@ class CAR(StrEnum):
   SANTA_FE_PHEV_2022 = "HYUNDAI SANTA FE PlUG-IN HYBRID 2022"
   SONATA = "HYUNDAI SONATA 2020"
   SONATA_LF = "HYUNDAI SONATA 2019"
+  STARIA_4TH_GEN = "HYUNDAI STARIA 4TH GEN"
```

Comment:
why 4th gen?

Reply:
The codename `US4` stands for 4th gen: https://thekoreancarblog.com/2023/10/14/hyundai-staria-hybrid-spied/#:~:text=Viewing%20the%20images%2C%20it%20could,it%20as%20an%20Hybrid%20vehicle.

![image](https://github.com/commaai/openpilot/assets/47793918/3aa3a759-5328-4a09-9f34-bf0658111b3d)